assembly/pom.xml

```diff
@@ -94,6 +94,11 @@
                                     <exclude>com.splicemachine:db-tools-ij</exclude>
                                     <exclude>com.splicemachine:db-shared</exclude>
                                     <exclude>com.splicemachine:db-drda</exclude>
+                                    <exclude>io.dropwizard.metrics:metrics-core</exclude>
+                                    <exclude>io.dropwizard.metrics:metrics-graphite</exclude>
+                                    <exclude>io.dropwizard.metrics:metrics-json</exclude>
+                                    <exclude>io.dropwizard.metrics:metrics-jvm</exclude>
+                                    <exclude>com.yammer.metrics:metrics-core</exclude>
```

Comment:
Please use io.dropwizard.metrics:* and com.yammer.metrics:*

Reply:
Ok